ci/steps/brew-bump.sh

```diff
@@ -13,27 +28,65 @@ main() {
   echo "Cloning cdrci/homebrew-core"
   git clone https://github.com/cdrci/homebrew-core.git
 
+  # Make sure the git clone step is successful
+  if [[ $(directory_exists "homebrew-core") -eq 1 ]]; then
+    echo "git clone failed. Cannot find homebrew-core directory."
+    ls -la
+    exit 1
+  fi
+
   echo "Changing into homebrew-core directory"
   cd homebrew-core && pwd
 
-  echo "Adding Homebrew/homebrew-core as $(upstream)"
+  echo "Adding Homebrew/homebrew-core"
   git remote add upstream https://github.com/Homebrew/homebrew-core.git
 
+  # Make sure the git remote step is successful
+  if ! git config remote.upstream.url > /dev/null; then
+    echo "git remote add upstream failed."
+    echo "Could not find upstream in list of remotes."
+    git remote -v
+    exit 1
+  fi
+
+  # TODO@jsjoeio - can I somehow check that this succeeded?
   echo "Fetching upstream Homebrew/hombrew-core commits"
   git fetch upstream
 
+  # TODO@jsjoeio - can I somehow check that this succeeded?
```

Comment:
you can likely check the exit status for `git merge` but this is likely a bit tricky to test, because the only time merges fail are when there's a merge conflict

Reply:
Ahh got it! Okay, maybe I'll just remove this line and ignore for now. I don't anticipate any merge conflicts since we're only ever updating our fork